File: client/mc/common2/src/main/java/calebxzhou/rdi/mc/common2/mcp/RMcpPlayerData.java
package calebxzhou.rdi.mc.common2.mcp;

public record RMcpPlayerData(
        String dim,
        String uuid,
        String name,
        RMcpPosData pos,
        Float health,
        Float maxHealth,
        Integer food,
        String gameMode,
        Integer latency
) {
}

File: mc/rmcp/common/src/main/kotlin/calebxzhou/rdi/mc/common2/mcp/RMcpPlayerData.kt
package calebxzhou.rdi.mc.common2.mcp

@JvmRecord
data class RMcpPlayerData(
    val dim: String,
    val uuid: String,
    val name: String,
    val pos: REntityPosData,
    val health: Float,
    val maxHealth: Float,
    val food: Int,
    val gameMode: String,
    val latency: Int?
)
